### PR TITLE
Updating blog documentation according to recent changes

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Signed-off-by: Pallavi-PH <pallaviph02@gmail.com>

In this PR we are updating blog documentation according to recent changes

### CODE REVIEW

**Review by IsAmrish — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 
-* Create a markdown file and add your blog content along with the metadata in    the following format
+* Create a markdown file and add your blog content along with the metadata in the following format
 ```
 ---
 title: Title of the blog
 author: Author name
 author_info: Author information
-tags: All the tags with space in between (Example: openebs localpv)
+tags: All the tags with comma separation in between (Example: openebs, localpv)
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md:11]: Should be OpenEBS, LocalPV

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 date: Date of publication in dd-mm-yyyy format
+excerpt: Short description about the blog in about 200 hundred words
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md:13]: Short description about the blog in about 200 hundred words. Words beyond 200 range will be clipped.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,7 +24,13 @@ Then, markdown file name will be ``title-of-the-blog.md``
 * The main blog image should be named as ``title-of-the-blog.png`` and add this file to ``website/website/public/images/blog``
 * If you are a new author, add your photograph with file name as `author-name.png` to `website/website/public/images/blog/authors`. Make sure the name provided in the author section of the metadata of the blog matches with this filename
 
+*Note:*
 
+* **Italic text:** To bold text, add two asterisks or underscores before and after a word or phrase. For example, `*italic text*`, `_italic text_`. Both of these texts will be rendered as *italic text*.
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md:29]: To bold text => To Italicize text

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -23,7 +24,13 @@ Then, markdown file name will be ``title-of-the-blog.md``
...
+* **Bold text:** To italicize text, add one asterisk or underscore before and after a word or phrase. For example, `**bold text**`, `__bold text__`. Both of these texts will be rendered as **bold text**
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md:30]: To italicize text => To make text bold

Please revisit these two notes regarding bold and italic

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -23,7 +24,13 @@ Then, markdown file name will be ``title-of-the-blog.md``
 * The main blog image should be named as ``title-of-the-blog.png`` and add this file to ``website/website/public/images/blog``
 * If you are a new author, add your photograph with file name as `author-name.png` to `website/website/public/images/blog/authors`. Make sure the name provided in the author section of the metadata of the blog matches with this filename
 
+*Note:*
 
+* **Bold text:** To bold text, add two asterisks or underscores before and after a word or phrase. For example, `**italic text**`, `__italic text__`. Both of these texts will be rendered as **bold text**.
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md:29]: `**italic text**`, `__italic text__` => `**bold text**`, `__bold text__`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -23,7 +24,13 @@ Then, markdown file name will be ``title-of-the-blog.md``
...
+* **Italic text:** To italicize text, add one asterisk or underscore before and after a word or phrase. For example, `*bold text*`, `_bold text_`. Both of these texts will be rendered as *italic text*
```
Comment (satyapriyamishra222) [inline CONTRIBUTING.md]: `*bold text*`, `_bold text_` => `*italic text*`, `_italic text_`

**Review by AVRahul — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**